src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,92 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private T[] elements = (T[]) new Object[DEFAULT_CAPACITY];
+    private int size;
+
+    public void resize() {
```

Comment:
```suggestion
    private void resize() {
```

I think we should make this method private (remember, private methods should follow after public as helper ones)